packages/react-components/src/components/chart/trendCursor/calculations/calculateSeriesMakers.ts

```diff
@@ -0,0 +1,65 @@
+// finding the left and right indexes for a given timestamp
```

Comment:
None of this is new logic right it's just moving around the structure?

Reply:
nope, just moved then to their own files. thats why they are not showing as file renamed